pass.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```

Comment:
![image](https://user-images.githubusercontent.com/43152234/124314309-7aec6700-db72-11eb-9978-6f93cc2fcddf.png)
Kiedy wprowadzę powtórzenie hasła tak, żeby pokrywało się z hasłem, to komunikat znika (co jest ok). Jednakże gdy zmienię to powtórzenie (lub hasło) w jakiś sposób tak, że te dwie wartości są różne, to komunikat nie wraca, a powinien :( Podobnie jest z pozostałymi komunikatami, np. gdy wprowadzę wielką literę a potem ją usunę